### DISCUSSION TITLE
Need help customising completions menu.

### DISCUSSION THREAD

**Discussion by asher-gh** [NvChad/NvChad#1125] [General]
I want to customise the completions menu only when I'm editing markdown files. The completions menu includes previous words. I just don't want the previous words in the completions menu.
<img width="880" alt="image" src="https://user-images.githubusercontent.com/74317567/170058238-9e6e5e79-00fa-4b54-a79f-2053a1edd0be.png">

**Comment by siduck** [comment]
thats buffer source, check cmp config and then override the sources table in your chadrc . so you want this behaviour only for markdown files or remove this feature?

**Reply by asher-gh** [reply]
only for markdown files

**Reply by siduck** [reply]
https://github.com/hrsh7th/nvim-cmp/issues/666
